distributed/protocol/serialize.py

```diff
@@ -195,6 +202,14 @@ def deserialize(self):
         frames = decompress(self.header, self.frames)
         return deserialize(self.header, frames)
 
+    def __eq__(self, other):
+        return (isinstance(other, Serialized) and
+                other.header == self.header and
+                other.frames == self.frames)
+
+    def __ne__(self, other):
+        return not (self == other)
```

Comment:
My reasoning was that if a `Serialized` object is somehow meant to represent the original object, then its hashability should also reflect the original object's hashability.

To be frank, I'm not sure what hashability really brings for `Serialize` and `Serialized` objects, but I'm happy to defer to your judgement.